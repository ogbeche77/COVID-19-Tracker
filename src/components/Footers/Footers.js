import React from 'react';
import styles from './Footers.module.css';
import { Typography } from '@material-ui/core';


function Footers() {
    return (
        <div>

            <Typography color="textSecondary" > &copy; {new Date().getFullYear()} Copyright:</Typography>
        </div>

    )
}

export default Footers