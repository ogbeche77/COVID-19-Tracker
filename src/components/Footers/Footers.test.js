/* eslint-disable no-undef */
import { shallow } from "enzyme";
import toJSON from "enzyme-to-json";
import React from "react";
import Footers from "./Footers";

test("matches snapshot", () => {
  const wrapper = shallow(<Footers />);
  expect(toJSON(wrapper)).toMatchSnapshot();
});
