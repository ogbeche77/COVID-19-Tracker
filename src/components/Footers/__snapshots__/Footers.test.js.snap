// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div>
  <WithStyles(ForwardRef(Typography))
    color="textSecondary"
  >
     
    Â© 
    2021
     Copyright:
  </WithStyles(ForwardRef(Typography))>
</div>
`;
